changelog.d/11286.doc

```diff
@@ -0,0 +1 @@
+Fix typo in the word `available` and fix HTTP method (should be `GET`) for the `username_available` admin API.
```

Comment:
We like to credit our contributors, you can include your nick or name if you'd like!

```suggestion
Fix typo in the word `available` and fix HTTP method (should be `GET`) for the `username_available` admin API. Contributed by Stanislav Motylkov. 
```

Reply:
Well, it technically should land in the git history, but ok, added it :slightly_smiling_face:

Reply:
Yep, but this is what we use to generate the changelogs, which are much more public!